src/main/java/com/tripply/Auth/entity/PasswordResetToken.java

```diff
@@ -0,0 +1,50 @@
+package com.tripply.Auth.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@Entity
+@Table(name = "password_reset_token", schema = "identity")
+public class PasswordResetToken {
+
+    private static final int EXPIRATION = 60 * 1;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    private String token;
+
+    @OneToOne(targetEntity = User.class, fetch = FetchType.EAGER)
+    @JoinColumn(nullable = false, name = "user_id")
+    private User user;
```

Comment:
Don't keep User table, as It will make mapping difficult, directly save userId instead of this.

Reply:
Resolved